CLAUDE.md

```
@@ -4,3 +4,10 @@
 
 - テストはファイル単位で実行する
     - 例: "rails test test/models/post_test.rb"
+- ルーティングのテストも書く
+    - test/routing/
+
+## コミット前の注意点
+
+- "bin/rubocop" のエラーがないことを確認すること
+    - エラーが発生したら "bin/rubocop -a" で修正を試みること
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)